### DISCUSSION TITLE
Delete all homes for specified User

### DISCUSSION THREAD

**Discussion by AdrianOliva2** [EssentialsX/Essentials#5918] [Ideas and Feature Suggestions]
### The command
/delhome <user>:*

### What does it do?
It deletes all the homes for a specified user.

So instead of having to do `/delhome <user>:<homeName>` every time until all homes are deleted, you can just do

`/delhome <user>:*`

And it'll delete all the user homes

### How is this useful?
Deleting all homes from a user as a punishment, or them losing rights to making them keep the extra homes still is not ideal for every server.`sethome.multiple`

Adding the feature to wildcard delete all the homes of a user (or any extra that go beyond their max home limit) would make managing temp homes useful, for example when you win a rank for a month and you get 3 more homes because it was set to do so, but after a month you no longer have those homes accessible.

**Comment by Evidentsinger14** [comment]
Discussions are not used for suggestions anymore. Suggest it in the [issue tracker](https://github.com/EssentialsX/Essentials/issues/new?assignees=&labels=type%3A+enhancement&projects=&template=request-a-feature.yml)
